recipe/meta.yaml

```diff
@@ -8,22 +8,23 @@ package:
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/eth_abi-{{ version }}.tar.gz
-  sha256: c3872e3ac1e9ef3f8c6599aaca4ee536d536eefca63a6892ab937f0560edb656
+  sha256: 33ddd756206e90f7ddff1330cc8cac4aa411a824fe779314a0a52abea2c8fc14
 
 build:
+  skip: True  # [py<39 or py>311]
```

Comment:
Because `parsimonious` does not have **py312+** support. Is it ok? Or is it better to rebuild `parsimonious`?

Reply:
According to upstream even py311 is not supported https://github.com/erikrose/parsimonious/blob/0.10.0/setup.py
But quick rebuild shows it works fine for py312. I think we can rebuild.